src/app/pages/form/simple-form/simple-form.component.scss

```diff
@@ -0,0 +1,26 @@
+.form {
+    position: relative;
+
+    tds-block {
+        transition: filter 250ms ease;
+    }
+
+    .spinner-container {
```

Comment:
I don't think we should hide this with CSS by default. I think a better approach would to conditionally render it in the DOM based on when the form is sending and not.